### DISCUSSION TITLE
TOC( Table of content file) : Allow One Plan into Multiple files

### DISCUSSION THREAD

**Discussion by sharma-TIP** [CMSgov/price-transparency-guide#294] [Q&A] [answered]
It would be much easier for everyone to associate multiple files with one file. This will allow to split the plan level information into multiple files and reduce redundancy, making the file sizes smaller.Plus the index files will show on what all files the plan information exists .  Any thoughts?

**Comment by Devyandu** [comment, marked as answer]
Agreed, it would very much help with plans who use a larger network from a national partner.

Similar discussion topic is [here](https://github.com/CMSgov/price-transparency-guide/discussions/260).

The [TOC PR ](https://github.com/CMSgov/price-transparency-guide/pull/291) and today's webinar mentioned that they want to keep each PR for only one issue. This PR was for file naming convention, the topic of multiple files is still being discussed internally (at CMS) and might get addressed in a future change.

PS : I am not with CMS.

**Reply by sharma-TIP** [reply]
Thanks @Devyandu ; Fingers crossed that multiple files are allowed. Everytime there is change our complete grouping logic changes which takes time to finalize , test and then run on a smaller set of data. We haven't started with a larger set of data which I presume will run into 10-50GB limits unless multiple files in TOC are allowed for a single plan.

**Reply by rajkumarsowmy** [reply]
yes, I do agree with this approach. because we cant accommodate all the data for a particular plan in 1 files, having a situation of plans home data + host data - it would go beyond 10-50GB compressed to present 50states pricing, Since most of us use AWS and BCBSA also mentioned the same technical difficulty and modified their filename with 1 0f 2 and 2 of 2 (in case of 2 files for a plan).  
so based on @Devyandu  suggestions on another post - we don't have to merge the files and have the references for each plan.
